.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
+max_line_length=88
+ignore = E501, W503, E203, F405, W605
```

Comment:
Can you teach your editor to end files with newlines? Causes unnecessary merge conflicts and diffs if we don't have that. That applies to most files in this diff.

Reply:
doh, thought this was handled by project settings, but not true.
fixed

Reply:
Done, all missing newlines should be fixed